src/profile-logic/processed-profile-versioning.js

```diff
@@ -2267,6 +2267,12 @@ const _upgraders = {
   [49]: (_) => {
     // The 'sanitized-string' marker schema format type has been added.
   },
+  [50]: (_) => {
+    // The serialized format now stores sample and counter sample times as
+    // time deltas instead of absolute timestamps to reduce the JSON size.
+    // The unserialized version is unchanged, and because the upgraders run
+    // after unserialization they see no difference.
```

Comment:
Same here - does version 50 require the delta format? The three places where this is documented should be consistent.